HabitTracker.Speedierone/HabitTracker.Speedierone/Program.cs

```diff
@@ -0,0 +1,242 @@
+﻿
+using Microsoft.Data.Sqlite;
+using System.Globalization;
+
+namespace Habit_Tracker;
+class Program
+{
+    static string connectionString = "Data Source=habit-Tracker.db";
+    static void Main(string[] args)
+    {
+        using (var connection = new SqliteConnection(connectionString))
+        {
+            connection.Open();
+
+            var tableCmd = connection.CreateCommand();
+
+            tableCmd.CommandText = @"CREATE TABLE IF NOT EXISTS drinking_water (
+                                    Id INTEGER PRIMARY KEY AUTOINCREMENT,
+                                    Date TEXT,
+                                    Quantity INTEGER)";
+
+            tableCmd.ExecuteNonQuery();
+
+            connection.Close();
+        }
+
+        GetUserInput();
+    }
+
+    static void GetUserInput()
+    {
+        Console.Clear();
+        bool closeapp = false;
+        while (closeapp == false)
+        {
+            Console.WriteLine("\nMain Menu");
+            Console.WriteLine("\nWhat would you like to do today");
+            Console.WriteLine("\nType 0 to exit programme");
+            Console.WriteLine("Type 1 to view all records");
+            Console.WriteLine("Type 2 to add a record");
+            Console.WriteLine("Type 3 to remove a record");
+            Console.WriteLine("Type 4 to update a record");
+
+            string command = Console.ReadLine();
+
+            switch (command)
+            {
+                case "0":
+                    Console.WriteLine("GoodBye");
+                    closeapp = true;
+                    Environment.Exit(0);
+                    break;
+                case "1":
+                    ViewAllRecords();
+                    break;
+                case "2":
+                    AddToRecord();
+                    break;
+                case "3":
+                    RemoveRecord();
+                    break;
+                case "4":
+                    UpdateRecord();
+                    break;
+                default:
+                    Console.WriteLine("Invalid Input");
+                    break;
+            }
+        }
+    }
+
+    private static void AddToRecord()
+    {
+        Console.Clear();
+        string date = GetDateInput();
+
+        int quantity = GetNumberInput("\n\nPlease insert number of glasses, no decimals allowed\n\n");
+
+        using (var connection = new SqliteConnection(connectionString))
+        {
+            connection.Open();
+            var tableCmd = connection.CreateCommand();
+            tableCmd.CommandText =
+                    $"INSERT INTO drinking_water(date, quantity) VALUES('{date}', {quantity})";
+
+            tableCmd.ExecuteNonQuery();
+            connection.Close();
+        }
+    }
+
+    internal static string GetDateInput()
+    {
+        Console.WriteLine("Please enter date in format dd-mm-yyyy");
```

Comment:
Looks like your parser is checking for dd-MM-yy but you're asking the user to enter dd-mm-yyyy, could you please fix that? 😁

Reply:
Heya, I have corrected this now and pushed it. I think it should have updated let me know if it hasnt.